tortoise/backends/asyncpg/client.py

```diff
@@ -200,11 +203,12 @@ def _in_transaction(self) -> "TransactionContext":
 
     async def create_connection(self, with_db: bool) -> None:
         await self._parent.create_connection(with_db)
-        self._connection = self._parent._connection
 
-    async def _close(self) -> None:
-        await self._parent._close()
-        self._connection = self._parent._connection
+    def acquire_connection(self) -> "ConnectionWrapper":
+        return ConnectionWrapper(self._connection, self._lock)
+
+    async def close(self) -> None:
```

Comment:
Is this an attempt to fix the reconnect test? If possible it would be best to not even have this?

Reply:
I mean the reconnect wrapper doesn't call `._close()` anymore.